draft-ietf-httpbis-sfbis.md

```diff
@@ -400,7 +401,7 @@ Strings are zero or more printable ASCII {{!RFC0020}} characters (i.e., the rang
 
 Unicode is not directly supported in Strings, because it causes a number of interoperability issues, and -- with few exceptions -- field values do not require it.
```

Comment:
```suggestion
Non-ASCII characters are not directly supported in Strings, because it causes a number of interoperability issues, and -- with few exceptions -- field values do not require it.
```